install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -745,11 +745,11 @@ ciliumEndpointSlice:
     limit: 10
     burst: 20
   - nodes: 100
-    limit: 7
-    burst: 15
+    limit: 50
+    burst: 100
   - nodes: 500
-    limit: 5
-    burst: 10
+    limit: 100
+    burst: 200
```

Comment:
Do these values for 500+ nodes actually work? Slightly above this diff is this warning:

```
  # limit: The sustained request rate in requests per second. The maximum rate that can be configured is 50.
  # burst: The burst request rate in requests per second. The maximum burst that can be configured is 100.
 ```

Reply:
Good catch, I forgot about the hard upper limit. I did some testing previously in raising that, but I'll revisit that later. For now I've just dropped the 500+ entry since it has no effect